exceotions.h

```diff
@@ -0,0 +1,25 @@
+/*****************************************************************************\
```

Comment:
I guess this file was added to the commit by mistake. Please remove it.

Reply:
Removed.